bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -41,7 +46,10 @@ sealed abstract class ITCRequest {
   def intParameter(name: String): Int = parameter(name).toInt
 
   /** Gets the named value as a double. */
-  def doubleParameter(name: String): Double = parameter(name).toDouble
+  def doubleParameter(name: String): Double = parameter(name).trim() match {
+    case ""   => 0.0
+    case s    => s.toDouble
+  }
```

Comment:
As a side note: all of that stuff is only used when accessing ITC through the web pages. Since my hope is that at some point in a not so far future all of this is replaced with something based on a decent modern web framework that has support for validation etc you could argue that it doesn't make sense to spend too much time on cleaning these things up. In the end it's not worse than what they had for the last 10 years... My main goal here was only to simplify things in a way that it is less verbose (e.g. use enum class names for the HTML parameter names and use the enum value names for the HTML option values etc).